### DISCUSSION TITLE
How does the titiler do memory release

### DISCUSSION THREAD

**Discussion by jason-jxf** [developmentseed/titiler#481] [Q&A]
![image](https://user-images.githubusercontent.com/24713773/170160376-53140800-9074-47b3-93b2-803273e02952.png)

I run the titiler in the container, and I found that as the number of images accessed increases, the memory will keep going up until the memory limit of the container, but there will be no OOM. Instead, it quickly declines after reaching the peak. What mechanism does this rely on to achieve?

**Comment by vincentsarago** [comment]
already mentioned in https://github.com/developmentseed/titiler/discussions/465

you can try using hypercorn instead of uvicorn/gunicorn
